assets/tests/hooks/useDetour.test.ts

```diff
@@ -169,7 +180,7 @@ describe("useDetour", () => {
   })
 
   test("when `waypoints` is not empty, `canUndo` is `true`", async () => {
-    const { result } = renderHook(useDetour)
+    const { result } = renderHook(() => useDetour("routePatternId"))
```

Comment:
**Nit (non-blocking):** This `renderHook(() => useDetour("routePatternId"))` is duplicated a lot throughout this test - I think in every single test in this file. Is there a way to extract it?

Reply:
f3509893